sunsynk/sunsynk.py

```diff
@@ -24,6 +25,8 @@ class Sunsynk:
     timeout: int = attrs.field(default=10)
     read_sensors_batch_size: int = attrs.field(default=60)
     timeouts: int = 0
+    dongle_serial_number: int = attrs.field(default=0)
```

Comment:
dongle_serial_number should ideally move to `solarmansunsynk.py`, as it is specific to that class

Reply:
I've made this change